Mathlib/Algebra/Order/LatticeGroup.lean

```diff
@@ -62,20 +62,9 @@ lattice, ordered, group
 -/
 
 
--- Needed for squares
 -- Needed for squares
 universe u
 
--- A linearly ordered additive commutative group is a lattice ordered commutative group
--- see Note [lower instance priority]
-@[to_additive]
-instance (priority := 100) LinearOrderedCommGroup.to_covariantClass (α : Type u)
-    [LinearOrderedCommGroup α] :
-    CovariantClass α α (· * ·)
-      (· ≤ ·) where elim a _ _ bc := OrderedCommGroup.mul_le_mul_left _ _ bc a
-#align linear_ordered_comm_group.to_covariant_class LinearOrderedCommGroup.to_covariantClass
-#align linear_ordered_add_comm_group.to_covariant_class LinearOrderedAddCommGroup.to_covariantClass
-
```

Comment:
Please add a `noalign` and/or porting note; or mark this PR as blocked by needing the SHA from the mathlib3 PR